app/views/foreman/smart_proxies/_content_tab.html.erb

```diff
@@ -28,7 +28,7 @@
     </tr>
     <tr ng-repeat-end ng-repeat="cv in env.content_views" ng-show="isEnvronmentExpanded(env)">
       <td>
-        <a href="/content_views/{{cv.id}}/versions" target="_self">
+        <a href="{{ fetchUrl(cv.default, cv.id) }}" target="_self">
```

Comment:
I think the name fetchUrl is a bit misleading since it doesn't fetch anything. Something like contentViewVersionUrl or productsOrVersionUrl are a bit clearer, or something else :)